WordPress/Classes/ViewRelated/Media/MediaLibraryPickerDataSource.m

```diff
@@ -532,6 +532,57 @@ - (WPMediaRequestID)imageWithSize:(CGSize)size completionHandler:(WPMediaImageBl
     return [self.mediaID intValue];
 }
 
+- (WPMediaRequestID)videoAssetWithCompletionHandler:(WPMediaAssetBlock)completionHandler
+{
+    if (!completionHandler) {
+        return 0;
+    }
+
+    NSString *errorMessage;
```

Comment:
Do you need this variable declared at this level? It could just be declared inside each of the if / guard statements?